extensions/wikia/CommunityPage/models/CommunityPageSpecialUsersModel.class.php

```diff
@@ -165,9 +165,12 @@ function () {
 			}
 		);
 
-		return $botIds;
+		return $botIds ?: [''];
 	}
 
+	/**
+	 * @return array
+	 */
 	private function getBotIds() {
 		$botIds = WikiaDataAccess::cache(
 			self::getMemcKey( self::ALL_BOTS_MCACHE_KEY ),
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)